config/application.rb

```diff
@@ -31,9 +31,19 @@ class Application < Rails::Application
       # Setup redis as the cache_store.
       # This is required for sidekiq, which uses redis to queue jobs.
       config.cache_store = begin
-        redis_config = Rails.application.config_for(:redis).symbolize_keys
+        redis_config = Rails.application.config_for(:redis).deep_symbolize_keys
+        opts = {}
+        if redis_config[:ssl_params]
+          opts.merge!({
+                        ssl: :true,
```

Comment:
Should this be `ssl: true`?

Reply:
it's taken straight from here: https://github.com/redis/redis-rb#ssltls-support
I hadn't tried a boolean value.